acct-creation-script/onboard.py

```diff
@@ -226,13 +230,13 @@ def get_assumed_session(environment, role):
     region_name = environment[1]
     boto_session = Session(region_name=region_name)
     sts = boto_session.client('sts', region_name=region_name) #If using opt-in regions, must have AWS_STS_REGIONAL_ENDPOINTS= 'regional' set
-    print(f'ID: {sts.get_caller_identity()}')
-    role_arn = f'arn:aws:iam::{environment[0]}:role/{role}'
-    assumed = sts.assume_role(RoleArn=role_arn, RoleSessionName='arpio_provisioning')
+    aws_account = environment[0]
+    role_arn = f'arn:aws:iam::{aws_account}:role/{role}'
+    assumed = dict(sts.assume_role(RoleArn=role_arn, RoleSessionName='arpio_provisioning'))
     assumed_session = Session(
-        aws_access_key_id=assumed['Credentials']['AccessKeyId'],
-        aws_secret_access_key=assumed['Credentials']['SecretAccessKey'],
-        aws_session_token=assumed['Credentials']['SessionToken'],
+        aws_access_key_id=str(assumed['Credentials']['AccessKeyId']),
```

Comment:
I think these will already be strings, right, and not need the `str()`?

Reply:
They should be